pkgs/development/python-modules/cymem/default.nix

```diff
@@ -20,7 +21,8 @@ buildPythonPackage rec {
     hash = "sha256-e4lgV39lwC2Goqmd8Jjra+znuCpxsv2IsRXfFbQkGN8=";
   };
 
-  propagatedBuildInputs = [
+  build-system = [
+    setuptools
     cython
```

Comment:
I thought for a moment that cython actually was a runtime dependency (I was confused about what the package does), but skimming through their code that doesn't appear to be the case